distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/ServiceManagerImpl.java

```diff
@@ -362,68 +287,55 @@ public void startBundle(String bundleSymbolicName) throws BundleException {
 
   @Override
   public void uninstallBundle(String bundleSymbolicName) throws BundleException {
-    for (Bundle bundle : getBundleContext().getBundles()) {
+    for (Bundle bundle : bundleContext.getBundles()) {
       if (bundleSymbolicName.equals(bundle.getSymbolicName())) {
         bundle.uninstall();
-        WaitCondition.expect(String.format("Bundle %s uninstalled", bundleSymbolicName))
-            .within(FEATURES_AND_BUNDLES_TIMEOUT, TimeUnit.MILLISECONDS)
+        await(String.format("Bundle %s uninstalled", bundleSymbolicName))
+            .atMost(FEATURES_AND_BUNDLES_TIMEOUT, TimeUnit.MILLISECONDS)
```

Comment:
✏️ Might want to leave a comment saying that the default poll interval is 100 ms just so other developers aren't confused because it's missing.

Reply:
If others think we should, i'll add a comment. But the previous API didn't specify a polling interval here either.